src/CustomerSupportService/appsettings.Development.json

```diff
@@ -0,0 +1,27 @@
+{
+  "RabbitMQHandler": {
+    "Host": "localhost",
+    "Username": "rabbitmquser",
+    "Password": "DEBmbwkSrzy9D1T9cJfa",
+    "Exchange": "Pitstop",
+    "Queue": "CustomerSupport",
+    "RoutingKey": ""
+  },
+  "ConnectionStrings": {
+    "CustomerSupportCN": "server=localhost,1434;user id=sa;password=8jkGh47hnDw89Haq8LN2;database=CustomerSupport;trustServerCertificate=true;"
```

Comment:
moeten we hier geen variables gebruiken?

Reply:
Wordt in de rest van Pitstop ook niet gedaan